transport/src/main/java/io/netty/bootstrap/Bootstrap.java

```diff
@@ -68,11 +70,11 @@ private Bootstrap(Bootstrap bootstrap) {
      * Sets the {@link NameResolver} which will resolve the address of the unresolved named address.
      */
     @SuppressWarnings("unchecked")
-    public Bootstrap resolver(NameResolverGroup<?> resolver) {
+    public Bootstrap resolver(AddressResolverGroup<?> resolver) {
```

Comment:
Could you explain why we switched from `AddressResolverGroup` to `NameResolverGroup`? A user can pass an unresolved address or a custom `SocketAddress` implementation to `Bootstrap.connect(SocketAddress)`.